dist/bom/controlplane-feature-sql-bom/build.gradle.kts

```diff
@@ -24,6 +24,7 @@ dependencies {
     api(project(":extensions:control-plane:store:sql:control-plane-sql"))
     api(project(":extensions:control-plane:store:sql:policy-definition-store-sql"))
     api(project(":extensions:control-plane:store:sql:transfer-process-store-sql"))
+    api(project(":extensions:control-plane:store:sql:participantcontext-store-sql"))
```

Comment:
I would not include the SQL variant in the SQL bom. When running EDC in a "single" participant context the store should not be required for operating and people can avoid managing an additional table

Reply:
sounds legit.